style.css

```diff
@@ -0,0 +1,460 @@
+* {
+  box-sizing: border-box;
+  color: white;
+  background-color: black;
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.inline {
+  display: inline;
+}
+
+li {
+  list-style: none;
+}
+
+/* this is the coloured border, will be removed - its only here because it helps me to visualise the layout */
+
+/* .icon,
+.border {
+    border: 1px dashed turquoise;
+} */
+
+/* this is the coloured border, will be removed */
+
+
+/* MEDIA QUERIES starts here */
+@media (max-width: 1260px) {
+  .right-panel {
+    width: 300px;
+  }
+}
+
+@media (max-width: 1180px) {
+  .right-panel {
+    display: none;
+  }
+}
+/* MEDIA QUERIES ends here */
+
+/* SCROLL BEHAVIOUS and HOVER starts here */
+::-webkit-scrollbar {
+  width: 0px;
+}
+
+.hover_round:hover {
+  background-color: var(--dark-grey);
+  transition: 0.3s;
+}
+
+.hover_square:hover {
+  background-color: var(--hover-square-grey);
+  transition: 0.3s;
+}
+
+.hover_square_more:hover {
+  background-color: var(--dark-grey);
+  transition: 0.3s;
+}
+
+.hover_blue:hover {
+  background-color: var(--hover-blue);
+  color: var(--hover-bright-blue);
+  border-radius: 50px;
+  padding: 5px;
+  transition: 0.3s;
+}
+
+.hover_green:hover {
+  background-color: var(--hover-green);
+  color: var(--hover-bright-green);
+  border-radius: 50px;
+  padding: 5px;
+  transition: 0.3s;
+}
+
+.hover_red:hover {
+  background-color: var(--hover-red);
+  color: var(--hover-bright-red);
+  border-radius: 50px;
+  padding: 5px;
+  transition: 0.3s;
+}
+
+.hover_content:hover {
+    background-color: rgba(8,8,8,255);
+    transition: 0.3s;
+}
+
+.hover_underline:hover {
+  text-decoration: underline;
+}
+
+.hover_content * {
+    background-color: transparent;
+}
+/* SCROLL BEHAVIOUS and HOVER ends here */
+
+/* COLOURS and FONT SIZES starts here*/
+html {
+  --twitter-blue: #1d9bf0;
+  --dark-grey: #15181c;
+  --grey-: #6e767d;
+  --light-grey: #eff3f4;
+  --hover-square-grey: #1c1f23;
+  --hover-blue: rgba(10, 23, 31, 255);
+  --hover-bright-blue: rgba(26, 136, 210, 255);
+  --hover-green: rgba(7, 26, 20, 255);
+  --hover-bright-green: rgba(4, 174, 115);
+  --hover-red: rgba(32, 9, 20, 255);
+  --hover-bright-red: #d0156b;
+}
```

Comment:
It made the work easier for me to group all the colours together before I started working on the colours. 
Therefore I did not have to think about the colour choices later.

Reply:
I like this, may start doing this myself if it really does make it easier in the long run

Reply:
i took a screenshot of twitter, then found all the colours I need even for the hovering all the shades of grey from there: https://imagecolorpicker.com/en .